### ISSUE TITLE
Rescheduling simplifications doesn't keep simplification bounds

### ISSUE THREAD

**Issue by QuackQuackBlah** [glinscott/fishtest#1086]

**Comment by QuackQuackBlah** [comment]
Sure, but what's the compelling reason that the default should be to change it to the normal LTC bounds, rather than keeping the simplification bounds for the test?

**Comment by dav1312** [comment]
Is this still an issue?
I rescheduled this test https://dfts-0.pigazzini.it/tests/view/62ef25a56f0a08af9f766ad5
and the bounds were kept https://dfts-0.pigazzini.it/tests/view/62f4ccb663129400ae5affa6

**Comment by vondele** [comment]
yes, looks like bounds are now kept (it still reads custom bounds though in the drop down menu, but they are right)

**Comment by peregrineshahin** [comment]
@vondele the correct bounds are always the custom bounds in the implementation as dav said IIRC, so chosing any bound would be read from the hidden custom bound if you wish, that's why changing custom bounds to new ones was necessary after changing them for the new bounds in the drop down list, it sounds a bit mihhh that they are not connect with one variable so maybe a PR to seperate them should be made, but if that's not an issue, this issue is resolved IMO

**Comment by vondele** [comment]
OK, I'm fine with closing.

**Comment by dubslow** [comment]
As of a month ago it was still a problem, I managed to run 30k LTC games at the wrong bounds. No idea about right now tho

**Comment by dav1312** [comment]
I'll work on a way to make it easier to realize if you have the wrong bounds but there are some PRs that I want to merge first
